docs/reference/cat/datafeeds.asciidoc

```diff
@@ -0,0 +1,155 @@
+[[cat-datafeeds]]
+=== cat {dfeeds} API
+++++
+<titleabbrev>cat {dfeeds}</titleabbrev>
+++++
+
+Returns configuration and usage information about {dfeeds}.
+
+[[cat-datafeeds-request]]
+==== {api-request-title}
+
+`GET /_cat/ml/datafeeds/<feed_id>` +
+
+`GET /_cat/ml/datafeeds`
+
+[[cat-datafeeds-prereqs]]
+==== {api-prereq-title}
+
+* If the {es} {security-features} are enabled, you must have `monitor_ml`,
+`monitor`, `manage_ml`, or `manage` cluster privileges to use this API. See
+<<security-privileges>> and {ml-docs}/setup.html[Set up {ml-features}].
+
+////
+[[cat-datafeeds-desc]]
+==== {api-description-title}
+
+TBD: This API returns a maximum of 10,000 {dfeeds}. 
+////
+
+[[cat-datafeeds-path-params]]
+==== {api-path-parms-title}
+
+`<feed_id>`::
+(Optional, string)
+include::{docdir}/ml/ml-shared.asciidoc[tag=datafeed-id]
+
+[[cat-datafeeds-query-params]]
+==== {api-query-parms-title}
+
+`allow_no_datafeeds`::
+(Optional, boolean)
+include::{docdir}/ml/ml-shared.asciidoc[tag=allow-no-datafeeds]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=http-format]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-h]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=help]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-s]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=time]
+
+include::{docdir}/rest-api/common-parms.asciidoc[tag=cat-v]
+
+[[cat-datafeeds-results]]
+==== {api-response-body-title}
+
+`assignment_explanation`::
+include::{docdir}/ml/ml-shared.asciidoc[tag=assignment-explanation]
++
+To retrieve this information, specify the `ae` column in the `h` query parameter. 
+
+`bucket.count`:: 
+include::{docdir}/ml/ml-shared.asciidoc[tag=bucket-count]
++
+To retrieve this information, specify the `bc` or `bucketCount` column in the
+`h` query parameter.
+
+`id`::
+include::{docdir}/ml/ml-shared.asciidoc[tag=datafeed-id]
++
+To retrieve this information, specify the `id` column in the `h` query parameter.
+  
+`node.address`::
+The network address of the node.
++ 
+include::{docdir}/ml/ml-shared.asciidoc[tag=node]
++
+To retrieve this information, specify the `na` or `nodeAddress` column in the
+`h` query parameter.
+  
+`node.ephemeral_id`::
+The ephemeral ID of the node.
++
+include::{docdir}/ml/ml-shared.asciidoc[tag=node]
++
+To retrieve this information, specify the `ne` or `nodeEphemeralId` column in
+the `h` query parameter.
+  
+`node.id`::
+The unique identifier of the node.
++
+include::{docdir}/ml/ml-shared.asciidoc[tag=node]
++
+To retrieve this information, specify the `ni` or `nodeId` column in the `h`
+query parameter.
+
+`node.name`::
+The node name.
++
+include::{docdir}/ml/ml-shared.asciidoc[tag=node]
++
+To retrieve this information, specify the `nn` or `nodeName` column in the `h`
+query parameter.
+
+`search.bucket_avg`::
+include::{docdir}/ml/ml-shared.asciidoc[tag=search-bucket-avg]
++
+To retrieve this information, specify the `sba` or `bucketTimeMin` column in the
```

Comment:
this is wrong. The API is also wrong as well. I will correct in a PR. Then we can correct it here :)

Reply:
This should be `searchBucketAvg`. Could you change it here and in https://github.com/elastic/elasticsearch/blob/master/x-pack/plugin/ml/src/main/java/org/elasticsearch/xpack/ml/rest/cat/RestCatDatafeedsAction.java#L95

Reply:
Good catch, thanks @benwtrent !  I've made that change